messagingmenu@screenfreeze.net/extension.js

```diff
@@ -258,9 +252,9 @@ const MessageMenu = new Lang.Class({
 			this.menu.addMenuItem(newLauncher);
 		}
 
-	},
+	}
 
-	_getApps: function() {
+    _getApps() {
```

Comment:
This changes how `_getApps()` needs to be called. The changes is probably correct, but the caller on line 149 will have to be updated accordingly as well. This is probably only the first of several such errors.